web/src/components/atoms/MarkDown/index.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "@emotion/styled";
+import { useRef, useState, FocusEvent } from "react";
+import ReactMarkdown from "react-markdown";
+
+import TextArea, { TextAreaProps } from "@reearth-cms/components/atoms/TextArea";
+
+type Props = {
+  value?: string;
+  onChange?: (value: string) => void;
+} & TextAreaProps;
+
+const MarkDownInput: React.FC<Props> = ({ value = "", onChange, ...props }) => {
+  const [showMD, setShowMD] = useState(true);
+  const textareaRef = useRef<HTMLInputElement>(null);
+
+  const handleBlur = (event: FocusEvent<HTMLTextAreaElement>) => {
+    event.stopPropagation();
+    setShowMD(true);
+  };
+
+  return (
+    <>
+      <TextArea
+        {...props}
+        onChange={e => onChange?.(e.target.value)}
+        onBlur={handleBlur}
+        value={value}
+        rows={6}
+        hidden={showMD}
+        ref={textareaRef}
+        showCount
+      />
+      <StyledMD
+        hidden={!showMD}
+        onClick={() => {
+          setShowMD(false);
+          if (textareaRef.current) {
+            setTimeout(() => {
+              textareaRef.current?.focus();
+            });
+          }
+        }}>
+        <ReactMarkdown>{value}</ReactMarkdown>
+      </StyledMD>
+    </>
```

Comment:
An atom should be a standalone component, so making it a fragment isn't really great. Please consider wrapping it in a div (it would be especially nice to have className passed in to this component and attached to the div to make this component more customizable by its parent component.

Reply:
Wrapped it with a `div` instead of the fragement